### DISCUSSION TITLE
The g++ downloaded in MYSY2 cannot compile the file. error with cc1plus

### DISCUSSION THREAD

**Discussion by Horc-zh** [msys2/MSYS2-packages#4952] [Q&A]
![image](https://github.com/user-attachments/assets/ad34f82a-ff32-4e1a-b926-37a21531861e)
It says the app(cc1plus) doesn't start properly.

when I type `which cc1plus` , I find there is no such app in my environment
![image](https://github.com/user-attachments/assets/07ad5429-698c-48f9-a347-e023c4eee391)

So ,I typed the command :
`pacman -S mingw-w64-x86_64-toolchain`

but from the output of  `which cc1plus`, it seems there is still no cc1plus in my current environment.
![image](https://github.com/user-attachments/assets/bfcb081c-9d38-4b9a-ae47-bfcfb4679103)

How can I solve this problem, thanks a lot.QAQ

**Comment by Biswa96** [comment]
Please follow the basic installation steps from here https://www.msys2.org/ and check if gcc works from the mingw64 or ucrt64 launcher.
